sfepy/discrete/conditions.py

```diff
@@ -29,6 +29,8 @@ def get_condition_value(val, functions, kind, name):
     elif (isinstance(val, Function) or nm.isscalar(val)
           or isinstance(val, nm.ndarray)):
         fun = val
+    elif isinstance(val, tuple):
```

Comment:
stylistic nitpicking: the other branches have an empty line above.

Would it make sense to allow lists as well? (See `is_sequence()` in `sfepy/base/base.py`).